CoreWidgetProvider/Widgets/SSHWalletWidget.cs

```diff
@@ -235,10 +235,11 @@ private JsonObject FillConfigurationData(bool hasConfiguration, string configFil
     {
         var configurationData = new JsonObject();
 
+        var defaultOrCurrentConfigFile = string.IsNullOrEmpty(configFile) ? DefaultConfigFile : configFile;
         var sshConfigData = new JsonObject
             {
                 { "configFile", configFile },
-                { "defaultConfigFile", DefaultConfigFile },
+                { "defaultOrCurrentConfigFile", defaultOrCurrentConfigFile },

```

Comment:
Not blocking, but this name seems awkward. Current config if one exists or default if it doesn't make sense, but the order here is reversed in the naming.